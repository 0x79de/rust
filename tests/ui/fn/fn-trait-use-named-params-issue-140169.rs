fn g(_: fn(a: u8)) {}
fn x(_: impl Fn(u8, vvvv: u8)) {} //~ ERROR expected one of `!`, `(`, `)`, `+`, `,`, `::`, or `<`, found `:`
fn y(_: impl Fn(aaaa: u8, u8)) {} //~ ERROR expected one of `!`, `(`, `)`, `+`, `,`, `::`, or `<`, found `:`
fn z(_: impl Fn(aaaa: u8, vvvv: u8)) {} //~ ERROR expected one of `!`, `(`, `)`, `+`, `,`, `::`, or `<`, found `:`

fn main(){}
