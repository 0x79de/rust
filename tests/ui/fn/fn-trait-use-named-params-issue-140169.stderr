error: expected one of `!`, `(`, `)`, `+`, `,`, `::`, or `<`, found `:`
  --> $DIR/fn-trait-use-named-params-issue-140169.rs:2:25
   |
LL | fn x(_: impl Fn(u8, vvvv: u8)) {}
   |                         ^ expected one of 7 possible tokens

error: expected one of `!`, `(`, `)`, `+`, `,`, `::`, or `<`, found `:`
  --> $DIR/fn-trait-use-named-params-issue-140169.rs:3:21
   |
LL | fn y(_: impl Fn(aaaa: u8, u8)) {}
   |                     ^ expected one of 7 possible tokens

error: expected one of `!`, `(`, `)`, `+`, `,`, `::`, or `<`, found `:`
  --> $DIR/fn-trait-use-named-params-issue-140169.rs:4:21
   |
LL | fn z(_: impl Fn(aaaa: u8, vvvv: u8)) {}
   |                     ^ expected one of 7 possible tokens

error: aborting due to 3 previous errors

